controllers/apply-next/simple/fields.js

```diff
@@ -1115,6 +1116,158 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                 .optional(),
             messages: []
         },
+        beneficiariesEthnicBakground: {
+            name: 'beneficiaries-ethnic-background',
+            label: localise({ en: `Ethnic background`, cy: '' }),
+            explanation: localise({
+                en: `You have indicated that your project mostly benefits people from a particular ethnic background, please select which one(s).`,
+                cy: ``
+            }),
+            type: 'checkbox',
+            optgroups: [
```

Comment:
`optgroups` isn't _technically_ correct for a name as it's not a real select `<optgroup>` but performs the same semantics (grouping options by a heading) so I kept the same name.